README.md

```diff
@@ -39,7 +39,7 @@ If your existing library (i.e. `go-ipfs` or `go-filecoin`) uses these other olde
 
 ```golang
 import (
-  graphsync "github.com/ipfs/go-graphsync"
+  graphsync "github.com/ipfs/go-graphsync/impl"
```

Comment:
uber-nit: the go community often uses `pkg` or even `internal` (inaccessible to other projects) for packages like this.

Reply:
I believe `pkg` is the dir used for the external packages. See https://github.com/golang-standards/project-layout

LGTM regardless.